Userland/Libraries/LibWeb/XHR/XMLHttpRequest.h

```diff
@@ -19,6 +19,18 @@
 
 namespace Web::XHR {
 
+static constexpr u8 horizontal_tab_byte = 0x09;
+static constexpr u8 line_feed_byte = 0x0A;
+static constexpr u8 carriage_return_byte = 0x0D;
+static constexpr u8 space_byte = 0x20;
+
+static constexpr Array<u8, 4> http_whitespace_bytes = {
+    horizontal_tab_byte,
+    line_feed_byte,
+    carriage_return_byte,
+    space_byte
```

Comment:
Can these be written as char literals instead? Would be just as clear as separate variables IMO. `'\t', '\n', '\r', ' '`